src/ol/webgl/shader.mustache

```diff
@@ -1,17 +1,17 @@
+/**
+ * @module {{{module}}}
+ */
 // This file is automatically generated, do not edit.
 // Run `make shaders` to generate, and commit the result.
 
-import {DEBUG_WEBGL} from '../../index.js';
-import _ol_webgl_Fragment_ from '../../webgl/Fragment.js';
-import _ol_webgl_Vertex_ from '../../webgl/Vertex.js';
-var {{namespace}} = {};
+import {DEBUG_WEBGL} from '../../../index.js';
+import _ol_webgl_Fragment_ from '../../../webgl/Fragment.js';
+import _ol_webgl_Vertex_ from '../../../webgl/Vertex.js';
```

Comment:
Might be worth renaming these while you're here.

Reply:
I've renamed the default exports in #7797.  But renaming these imports would be easier done in your branch.

Reply:
I've added the renaming to this branch